RawSpeed/ArwDecoder.cpp

```diff
@@ -63,6 +63,40 @@ RawImage ArwDecoder::decodeRawInternal() {
       }
 
       return mRaw;
+    } else if (hints.find("srf_format") != hints.end()) {
```

Comment:
Could we give this a value, so we can recognize different variations in the future?

If there is no good name, "v1" would be fine.